src/Post/PostPolicy.php

```diff
@@ -99,6 +99,7 @@ public function find(User $actor, $query)
                             ->from('discussions')
                             ->whereColumn('discussions.id', 'posts.discussion_id')
                             ->where(function ($query) use ($actor) {
+                                $query->whereRaw('1=0');
```

Comment:
Could you try whether this would work as well?

~~~php
->where(function ($query) use ($actor) {
  $query->whereRaw('1=0')
    ->orWhere(function ($query) use ($actor) {
      $this->events->dispatch(...);
    });
});
~~~

Because if I understand [Laravel's implementation](https://github.com/laravel/framework/blob/3ec56b8810bc40016252977c69ba27057dcecf7d/src/Illuminate/Database/Query/Builder.php#L1361-L1372) correctly, the `orWhere()` would not be applied if none of the event listeners adds a  WHERE clause to the query.

Reply:
The one test that registers a listener for this event could then also use `where()` instead of `orWhere()`.

Reply:
Assuming you meant to replace the whereRaw, this doesn't work. If you wanted to use orWhere after the whereRaw, it doesn't add anything, it only makes the code less readable.

Reply:
However setting the test to use `whereRaw` instead of `orWhereRaw` does make a difference. Committing.